toasty/simple_interface.py

```diff
@@ -0,0 +1,51 @@
+from toasty import builder, collection, pyramid, multi_tan, multi_wcs
+import reproject
+
+
+def toast_fits_list(fits_list, **kwargs):
+    """
+    Process a file or a list of FITS files into a tile pyramid with
+    a common tangential projection.
+
+    Parameters
+    ----------
+    fits_list : str or list of str
+        A single path or a list of paths to FITS files to be processed.
+    kwargs
+        Settings for the `toasty` tiling process. Common
+        settings include 'hdu_index' and 'blankval'.
+
+    Returns
+    -------
+    out_dir : :class:`str`
+        The relative path to the base directory where the tiled files are located
+    image_set_pattern : :class:`str`
+        The pattern to be used to access specific tiles
+    """
+
+    if not isinstance(fits_list, list):
+        fits_list = [fits_list]
+
+    out_dir = fits_list[0].split('.')[0]
+
+    pio = pyramid.PyramidIO(out_dir, default_format='fits')
+    bld = builder.Builder(pio)
+    coll = collection.SimpleFitsCollection(fits_list, **kwargs)
+    print('Processing FITS - Step 1 of 2')

```

Comment:
This function should take `cli_progress` as a keyword argument and only print these messages if its value is truthy.

I think it might also be good to have more descriptive names for these steps, rather than numbering; something like "tiling base layer"  and "downsampling" rather than "step 1" and "step 2".

Reply:
I've updated the texts now, to "Tiling base layer - (Step 1 of 2)". I think it's a good idea to be upfront with the fact that there are two steps and two progress bars. It can be very demoralizing to see a slow progress bar finish, just to be replaced with a new progress bar.